KEYS

```diff
@@ -467,3 +467,38 @@ w3cV0dFYUrtUKqCyivhqu2YP5kV15nFkDTg/Q1lNuGUTsqlgNh4dFDAeY4/1eGEU
 ynWT9tKikBHrwNOMsSnbkdQwukw2OnUdY4md2gbhxaRmF5U3KwRp7v++
 =loDK
 -----END PGP PUBLIC KEY BLOCK-----
+
+pub   rsa4096 2025-04-01 [SC]
+      24E1775CC44ED9CED4CF03672F0FBAA76B492842
+uid           [ unknown] Apache Daffodil Automated Repease Signing <private@daffodil.apache.org>
```

Comment:
Repease typo

Reply:
That typo is in the key, I mention it to ASF infra and see if they can regenerate a new one

Reply:
We got a new key from ASF INFRA with the typo fixed, PR updated.